### ISSUE TITLE
Sources default to custom sources after reinstalling

### ISSUE THREAD

**Issue by KobeW50** [ReVanced/revanced-manager#897]
### Type

Error while running the manager

### Bug description

I noticed after uninstalling and reinstalling the manager (1.3.0), my sources were somehow set to inotia00 (revanced extended) in the new manager, despite it not having been set that at the time when I uninstalled the previous manager. 

Here's what I did:

Reset the sources. Closed the app. Reopened it to ensure the sources were indeed reset.

Then I cleared the data/cache.

Uninstalled the manager. (Which was 1.3.0, the second release)

Powered off my phone. Then powered on my phone.

Cleared up my ram using the clear button in the samsung device care settings

Downloaded and installed a new manager apk from github.

Opened it and it still defaults to inotia00 

### Steps to reproduce

I dont know the step that caused this to start in the first place. But here goes:

Use custom sources (I guess)

Reset the sources and restart the app.

Uninstall the app.

Reinstall the app.

### Android version

13

One UI 5.1

S22+

### Manager version

1.3.0 (second release)

### Target package name

NA

### Target package version.

NA

### Installation type

Non-root

### Patches selected.

NA

### Device logs (exported using Manager settings).

```shell
Check comment bellow
```

### Screenshots or videos

https://github.com/revanced/revanced-manager/assets/84587632/9825c29a-e4b3-4298-b77f-d469e2229803

### Additional context

I use custom sources frequently. It may have been set to inotia00 when I updated the manager from 1.2.0 to 1.3.0 (first release).

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.
- [X] The issue is related solely to the ReVanced Manager and not related to patching errors or patches

**Comment by KobeW50** [comment]
![revanced-manager_logcat_20230525001957413955.log.txt](https://github.com/revanced/revanced-manager/files/11561290/revanced-manager_logcat_20230525001957413955.log.txt)

**Comment by SodaWithoutSparkles** [comment]
The problem is removing an app in android _sometimes_ dont remove all of its data. So manager dont know if the data is just set right now of leftovers from previous installs

**Comment by TheAabedKhan** [comment]
@KobeW50 are you facing this in the latest version of the manager as well?

**Comment by KobeW50** [comment]
> @KobeW50 are you facing this in the latest version of the manager as well?

Haven't had the issue in a while now. Probably should've updated this issue but I remember at one point it seemed like the issue was resolved but then it occurred again after.

I still don't know what the root cause of the issue is. It may only occur after patching with custom sources once (or twice, like the patched applications bug).

I haven't used custom sources with the Manager in a long time. Just now I tried testing by just changing the sources and then reinstalling and the issue wasn't apparent. Ig this can be closed for now. I'm too lazy to do thorough testing. 

Thank you for helping make the Manager amazing
